docs/tutorial/getting-started.md

```diff
@@ -82,14 +112,63 @@ unit-jenkins-k8s-0:
     started: <timestamp>
 ```
 
-You can now access your Jenkins server UI at `http://<UNIT_IP>:8080` and login using username
-"admin" and password from the action above.
+You can now access your Jenkins server UI at the previous url, and login using username "admin" and password from the action above.
+
+### Deploy and integrate k8s agents
+
+By default, jenkins-k8s server application is installed with 0 executors for security purposes.
+A functional Jenkins application requires additional Jenkins agents to be integrated.
+
+The following commands deploy 3 units of the jenkins-agent-k8s charm and integrate them with the
+jenkins-k8s charm.
+
+```
+juju deploy jenkins-agent-k8s --channel=latest/edge --num-units=3
+
+# 'agent' relation name is required since jenkins-k8s charm provides multiple compatible
+# interfaces with jenkins-agent-k8s charm.
+juju integrate jenkins-k8s:agent jenkins-agent-k8s:agent
+```
+
+### Deploy and integrate machine agents (optional)
+
+For this section you need a machine model named `lxd`. 
```

Comment:
Maybe mention that the Multipass blueprint will automatically set up the machine model for you